config.toml

```diff
@@ -2,6 +2,8 @@ baseURL = "https://docs.kubermatic.com/"
 theme = "hugo-theme-learn"
 languageCode = "en-us"
 title = "Kubermatic Docs"
+disableKinds = ["taxonomy"]
+ignoreErrors = ["error-disable-taxonomy"]
```

Comment:
What's this about? Why do we need to ignore errors?

Reply:
@xrstf 
First line is prevent of generation for these useless (at current moment) pages:
https://docs.kubermatic.com/categories/
https://docs.kubermatic.com/tags/

Second line is needed for disable an error, that appears on build. Seems like Hugo expect taxonomy items by default. Also, some naming is changed for taxonomies from [v0.73](https://gohugo.io/news/0.73.0-relnotes/). The Hugo version was updated to 0.75 in this PR as well.